packages/base/room.gts

```diff
@@ -576,26 +549,30 @@ export class RoomField extends FieldDef {
             ...cardArgs,
             attachedCardIds,
           });
-        } else if (event.content.msgtype === 'org.boxel.command') {
+        } else if (
+          event.content.msgtype === 'org.boxel.command' &&
+          event.content.data.toolCall
```

Comment:
1 thing to point out about this PR. We have altered the structure of the command event `org.boxel.command` in matrix so we just filter out all commands messages with the old structure. Otherwise, the host app will break with `recompute` errors.